site/gatsby-site/migrations/2022.02.18T16.29.14.increment-report-number.js

```diff
@@ -0,0 +1,39 @@
+const lodash = require('lodash');
+
+/**
+ *
+ * @param {{context: {client: import('mongodb').MongoClient}}} context
+ */
+
+exports.up = async ({ context: { client } }) => {
+  await client.connect();
+
+  const collection = client.db('aiidprod').collection('incidents');
```

Comment:
In the future, these should likely be loaded from config. No need to change them at the moment.